app/helpers/footer_helper.rb

```diff
@@ -28,13 +28,13 @@ def show_authors_and_editors(obj)
       if is_admin
         authors += safe_nbsp
         authors += link_with_query("(#{:review_authors_review_authors.t})",
-                                   controller: :observer,
+                                   controller: :authors,
                                    action: :review_authors,
                                    id: obj.id, type: type)
       elsif !is_author
         authors += safe_nbsp
         authors += link_with_query("(#{:show_name_author_request.t})",
-                                   controller: :observer,
+                                   controller: :authors,
```

Comment:
Is it worth removing the redundant "review" from these two actions?

Long, long ago Nathan and I decided the retain the redundancy -- thus name/show_name, image/show_image, etc.  But that clearly is not the CRUD Way™.  So since we're refactoring anyway, we might as well consistently remove the redundancy throughout, right?  Or does this add too much work?  (Such as changing rss_log to activity_log would have done.)

Reply:
That's right, let's keep it on-brand! The CRUD Way™ hahaha

But i guess you meant the redundant "author"... oder?

Reply:
Update: `request` seems to be a reserved word. So i made it `authors/email_request`